Fabric.Identity.API/Quickstart/Account/AccountController.cs

```diff
@@ -258,6 +258,7 @@ public async Task<IActionResult> ExternalLoginCallback(string returnUrl)
             }
             catch(InvalidIssuerException exc)
             {
+                await HttpContext.Authentication.SignOutAsync(IdentityServerConstants.ExternalCookieAuthenticationScheme);

```

Comment:
Is this the only cookie that is set at this point that we have to delete?

Reply:
I believe so - I deleted the same cookie that is deleted at the end of this method after user login succeeds.